queries.sql

```diff
@@ -12,7 +12,46 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 
 SELECT * FROM animals WHERE neutered = true;
 
-SELECT * FROM ANIMALS WHERE name NOT LIKE '%Gabumon%';
+SELECT * FROM animals WHERE name NOT LIKE '%Gabumon%';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+UPDATE animals SET species = 'unspecified' WHERE species IS NULL;
+
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
```

Comment:
- [ ] After updating your table, as required in the project you should delete all records in the `animals` table, then roll back the transaction 😁